middleware/getPhotoOfTheDay.js

```diff
@@ -0,0 +1,16 @@
+const fetch = require("node-fetch");
+const API_KEY = require("../apiKey");
+const BASE_URL = "https://api.nasa.gov/planetary/apod";
+
+const getPhotoOfTheDay = async (request, response, next) => {
+  const url = `${BASE_URL}?api_key=${API_KEY}`;
+  try {
+    const apodResponse = await fetch(url);
```

Comment:
What happens if NASA's API is down or if there's a problem with our code somewhere?

Does our server return a response?

We'd ideally like our endpoint to give some sort of error code. Let's start by a blanket decision that if there's a problem, our endpoint should return a response with HTTP status code 500.

Reply:
I've setup some error handling, let me know what you think :)